base/system/userinit/lang/es-ES.rc

```diff
@@ -23,24 +28,24 @@ CAPTION "ReactOS LiveCD"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "IDB_LOGO", IDC_STARTLOGO, "Static", WS_CHILD | WS_VISIBLE | SS_OWNERDRAW, 18, 4, 290, 99
-    DEFPUSHBUTTON "Run ReactOS &Live CD", IDC_RUN, 71, 108, 175, 21
-    PUSHBUTTON "&Install ReactOS", IDC_INSTALL, 71, 136, 175, 21
+    DEFPUSHBUTTON "Iniciar &Live CD de ReactOS", IDC_RUN, 71, 108, 175, 21
+    PUSHBUTTON "&Instalar ReactOS", IDC_INSTALL, 71, 136, 175, 21
 
     // LTEXT "", IDC_STATIC, 7, 152, 300, 8
 
     CONTROL "", IDC_STATIC, "STATIC", SS_ETCHEDHORZ, 0, 165, 317, 1
-    PUSHBUTTON "&Previous", IDOK, 205, 172, 50, 14
-    PUSHBUTTON "&Cancel", IDCANCEL, 260, 172, 50, 14
+    PUSHBUTTON "&Anterior", IDOK, 205, 172, 50, 14
+    PUSHBUTTON "&Cancelar", IDCANCEL, 260, 172, 50, 14
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SHELL_FAIL "Userinit no pudo iniciar el shell con éxito!"
-    IDS_INSTALLER_FAIL "Userinit failed to start the installer!"
-    IDS_CANCEL_CONFIRM "Are you sure you want to quit the ReactOS LiveCD?\nIf you choose to do so, your computer might restart."
+    IDS_SHELL_FAIL "Userinit no pudo iniciar el shell con éxito."
+    IDS_INSTALLER_FAIL "Userinit no pudo iniciar el instalador."
```

Comment:
These sentences end with an exclamation sign in other translations:
- https://git.reactos.org/?p=reactos.git&a=search&h=master&st=grep&s=IDS_SHELL_FAIL
- https://git.reactos.org/?p=reactos.git&a=search&h=master&st=grep&s=IDS_INSTALLER_FAIL